jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/logging/ConsoleLoggerBuilder.java

```diff
@@ -39,11 +39,15 @@
    *
    * @param singleThreadedExecutor a {@link SingleThreadedExecutor} to ensure that all messages are
    *     logged in a sequential, deterministic order
+   * @param mavenFix fixes logging issue in Maven
+   *     (https://github.com/GoogleContainerTools/jib/issues/1952)
    * @return a new {@link ConsoleLoggerBuilder}
    */
-  public static ConsoleLoggerBuilder rich(SingleThreadedExecutor singleThreadedExecutor) {
+  public static ConsoleLoggerBuilder rich(
+      SingleThreadedExecutor singleThreadedExecutor, boolean mavenFix) {
```

Comment:
I don't know about this parameter name, maybe we should just have two different `richGradle` and `richMaven` methods instead of passing in a `boolean`. Although I'm not sure if I really like that more. It at least sounds less hacky (then again I guess this is a bit of a hack, and maybe we shouldn't try to hide that).

Reply:
Yeah, agreed. Kinda expected this reaction. I wanted to avoid putting a specific builder name, since theoretically we may have more builders in the future. I think I just came up with a much better name: `enableTwoCursorUpJump`. The rich console is all about the hacky cursor trick, so this kind of console-controlling flag names for the constructor method sound pretty natural to me.